apps/spruce/src/pages/projectSettings/tabs/EventLogTab/EventLogTab.tsx

```diff
@@ -25,6 +28,13 @@ export const EventLogTab: React.FC<TabProps> = ({ limit, projectType }) => {
   return (
     <EventLog
       allEventsFetched={allEventsFetched}
+      customKeyRenderConfig={{
+        "vars.vars": (key) => (
+          <StyledInlineDefinition definition="Evergreen does not display project variable values.">
```

Comment:
IMO this isn't really offering the user any additional context. Maybe something like this?
```suggestion
          <StyledInlineDefinition definition="Evergreen does not display project variable values in the event log for security reasons.">
```